from django.core.exceptions import ValidationError as DjangoValidationError
from rest_framework.exceptions import ValidationError as SerializerValidationError
